docs/configuration.rst

```diff
@@ -344,6 +344,22 @@ Traces
      version_added:
         v1.9.0:
 
+   DD_TRACE_SAFE_INSTRUMENTATION_ENABLED:
+     type: Boolean
+     default: False
```

Comment:
Will the SSI harness be updated to set this variable to True, or will the default be used?

Reply:
The SSI harness will set it to true.